ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -46,12 +47,13 @@ type LifeCycleServiceServer struct {
 }
 
 const (
-	hostVportId = "03"
-	accVportId  = "04"
-	deviceId    = "0x1452"
-	vendorId    = "0x8086"
-	imcAddress  = "192.168.0.1:22"
-	apfNumber   = 16
+	hostVportId         = "03"
+	accVportId          = "04"
+	deviceId            = "0x1452"
+	vendorId            = "0x8086"
+	imcAddress          = "192.168.0.1:22"
+	apfNumber           = 16
+	last_byte_mac_range = 246
```

Comment:
https://github.com/intel-innersource/drivers.ethernet.linux.mev-cp/blob/7edcb25af59a930de8e932ba095bd8668e765b00/src/mac_utils.c#L181

ipu_platform/imc/agents/userspace/mev_imc_dpcp_src/src/libdpcp/src/mac_utils.c

81:  base_mac_addr[5] = policy mac_addr[5] + func_id

The 6th byte of base mac addr is set for all 16 function ids and the randomization should be in slot of 16 mac addresses.

last_byte_mac_range should be in the range of 256-16 = 240.